java/libphonenumber/src/com/google/i18n/phonenumbers/Phonemetadata.java

```diff
@@ -0,0 +1,1119 @@
+/*
+ * Copyright (C) 2010 The Libphonenumber Authors
```

Comment:
What do you mean? Which file *is* it based off?
I mentioned in the commit that it's "almost what it was in 7.1.1", do you mean phonemetada.proto? I didn't generate it from that, I just rolled back and added new API by hand (mentioned in commit).